packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/en.ftl

```diff
@@ -4,4 +4,5 @@ cadReminderFirst-action-plaintext = { cadReminderFirst-action }:
 # In the title of the email, "It takes two to sync", "two" refers to syncing two devices
 cadReminderFirst-title-1 = It takes two to sync
 cadReminderFirst-description-1 = Take your tabs across all your devices. Get your bookmarks, passwords, and other data everywhere you use { -brand-firefox }. It’s like having magic in your { -brand-firefox } account!
+cadReminderFirst-description-part1-v2 = Take your tabs across all your devices. Get your bookmarks, passwords, and other data everywhere you use { -brand-firefox }.
```

Comment:
Why `-v2` instead of `-2` postfix? Is it because the l10n id already ends with a number?